pkgs/by-name/ra/rastertoezpl/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cups,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rastertoezpl";
+  version = "1.1.12";
+
+  src = fetchurl {
+    url = "https://godex.s3-accelerate.amazonaws.com/tWHDYruxWFM98frjWNTE,g.file?v01";
+    name = "${pname}-${version}-source.tar.gz";
+    hash = "sha256-iBhM8Mht/XncWU75cd485WK5GZtJNv78yMsFsD0eKWQ=";
+  };
+
+  buildInputs = [ cups ];
+
+  configureFlags = [
+    "--datarootdir=${placeholder "out"}/share"
+    "PPDDIR=${placeholder "out"}/share/cups/model"
+  ];
+
+  postInstall = ''
+    mkdir -p $out/lib/cups/filter
+    ln -s $out/libexec/rastertoezpl/rastertoezpl $out/lib/cups/filter/rastertoezpl
+  '';
+
+  meta = {
+    description = "CUPS driver for GODEX printers";
+    homepage = "https://www.godexintl.com/downloads";
+    license = lib.licenses.gpl2;
```

Comment:
Licenses like `gpl2` are deprecated as it's not clear if they *include* or *exclude* newer versions, see https://github.com/NixOS/nixpkgs/pull/107417 .  Please replace this with `gpl2Plus` which seems to be right choice from looking into the source tarball.